vsphere/datadog_checks/vsphere/metrics.py

```diff
@@ -64,6 +64,13 @@
     'vsanDomObj.readCacheHitRate.latest',
 }
 
+VSAN_PERCENT_METRICS = {
+    'congestion',
+    'coreUtilPct',
+    'pcpuUsedPct',
+    'pcpuUtilPct',
+}
```

Comment:
nit- can we use the full metric name so it's similar to `PERCENT_METRICS`?

Reply:
Done.